build.sbt

```diff
@@ -1,16 +0,0 @@
-
-organization := "org.kframework.k"
-
-name := "kore"
-
-scalaVersion := "2.12.2"
```

Comment:
I'll put `build.sbt` into a new branch where we can make sure it works on everyone's system before merging it into master.